### PR TITLE
added blog post about first two weeks as intern

### PR DESCRIPTION
**Pull request by kdelee**
Let me know what you think!

### CODE REVIEW

File: _posts/2017-06-22-pulp-qe-intern-update.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: 2017 Pulp QE Intern Update
+author: Elijah DeLee
+tags:
+  - pulpqe-intern
```
Comment (dkliban) [inline _posts/2017-06-22-pulp-qe-intern-update.md]: This tag does not exist. If you want to add a new tag, you need to add it in _data/tags.yml and tags/tag_<tagname>.html

A better tag to create would be 'pulpqe'

**Review by dkliban — CHANGES_REQUESTED** [review]

**Review by dkliban — APPROVED** [review]
Thanks!

File: _posts/2017-06-22-pulp-qe-intern-update.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: 2017 Pulp QE Intern Update
+author: Elijah DeLee
+tags:
+  - pulpqe
+---
+
+### Virtualization, docker, automated tests and more!
```
Comment (dkliban) [inline _posts/2017-06-22-pulp-qe-intern-update.md]: remove ### and add -------------------------- on the next line. Do the same for each header. here is an example https://raw.githubusercontent.com/pulp/pulpproject.org/gh-pages/_posts/2016-12-05-contributing-blog-post-to-pulpproject.org.md

**Comment by kdelee** [comment]
@dkliban 
I fixed the headers, thanks for looking at this

**Pull request merged**